examples/web_view.rs

```diff
@@ -152,6 +152,8 @@ fn start_example_client_code() -> Result<()> {
             "-an",
             "-c:v",
             "libx264",
+            "-profile:v",
```

Comment:
verify if this works without this

Reply:
Removing this, committed it accidentally